readme.md

```diff
@@ -125,18 +125,24 @@ _Optional, Default: `INFO`_
 
 Set `LogLevel` for detailed information about plugin operation.
 
-##### `url`
+##### `serverList.url`
 
 _Required, Default: `""`_
 
 LDAP server address where queries will be performed.
 
-##### `port`
+##### `serverList.port`
 
-_Optional, Default: `389`_
+_Required, Default: `389`_
 
 LDAP server port where queries will be performed.
 
+##### `serverList.port`
```

Comment:
I think you meant to write serverList.weight

Reply:
You are right. Fixed.